### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by gfournieriExec**

### CODE REVIEW

File: JenkinsfileBuild
Comment (zguesmi) [inline JenkinsfileBuild:1]: Can we rename them to `build.jenkinsfile` and `deploy.jenkinsfile` to keep the IDE detection of the language? (not sure if that works to be honest)

Comment (gfournieriExec) [reply JenkinsfileBuild:1]: Could be an idea yes, but my IDE still detect it as Jenkinsfile language 
image

**Review by jeremyjams — APPROVED** [review]
Ok thanks :ok_hand:

File: JenkinsfileBuild
Comment (jeremyjams) [reply JenkinsfileBuild:1]: Language is Groovy :)

Comment (gfournieriExec) [reply JenkinsfileBuild:1]: indeed, changing the name won't let spot it as Jenkinsfile (with the logo) neither actually 
image

Comment (jeremyjams) [reply JenkinsfileBuild:1]: https://github.com/jenkinsci/pipeline-examples

Comment (zguesmi) [reply JenkinsfileBuild:1]: Ok we can keep it that way.
Thanks for checking.

File: JenkinsfileDeploy [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            sh """
+                docker run \
+                    --rm \
+                    -e NETWORK_NAME='$userInput.networkName' \
+                    -e VOUCHER_HUB_ADDRESS='$userInput.voucherHubAddress' \
+                    -e VOUCHER_HUB_START_BLOCK='$userInput.voucherHubStartBlock' \
+                    -e IPFS_URL='http://$userInput.targetRemoteHost:5001' \
+                    -e GRAPHNODE_URL='http://$userInput.targetRemoteHost:8020' \
+                    -e VERSION_LABEL='$userInput.versionLabel' \
+                    docker-regis.iex.ec/voucher-subgraph-deployer:$userInput.dockerImageVersion
```
Comment (zguesmi) [inline JenkinsfileDeploy:62]: Why do we push to `dockerhub` but for deployment we pull from `docker-regis`?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,103 @@
-# iexec-voucher-subgraph
-Subgraph for iExec Voucher contracts
+# voucher-subgraph
+
+A subgraph indexing `iexec-voucher-contracts`
```
Comment (zguesmi) [inline README.md:3]: We can put a link to the contracts repo.

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# iexec-voucher-subgraph deployer
+
+FROM node:18
```
Comment (zguesmi) [inline docker/Dockerfile:3]: Why not a more recent version?

Comment (gfournieriExec) [reply docker/Dockerfile:3]: That's the dockerfile given by Pierre directly. So I did not touched it, could be a 1.0.1 version I guess

Comment (gfournieriExec) [reply docker/Dockerfile:3]: He already tested that version so I don't think changing the node version rn is the good option

File: JenkinsfileDeploy (hunk 1 above)
Comment (gfournieriExec) [reply JenkinsfileDeploy:62]: Will be done in a Next PR

File: README.md (hunk 2 above)
Comment (gfournieriExec) [reply README.md]: Done

**Review by zguesmi — APPROVED** [review]

**Pull request merged**